Start post

ivelasq opened the pull request.

ivelasq committed:
Start post

ivelasq committed:
Optimize images

ivelasq committed:
Fix tab2 function call

gvelasq commented on blog/not-basic-base-r-functions/index.qmd:
```diff
@@ -48,7 +48,7 @@ Many of your favorite functions use `invisible()`. For example, `readr::write_cs
 If you're writing functions, then `invisible()` can be very helpful. 
 
-* In the example below, using {purrr} and `tidytab::tab_run())` together results in two `NULL` list items that I do not need. 
+* In the example below, using {purrr} and `tidytab::tab_run()` together results in two `NULL` list items that I do not need. 
```
Should this be `tidytab::tab2()`?

gvelasq replied:
😅

ivelasq committed:
Continue post

ivelasq committed:
Add headers for webr

ivelasq committed:
Rerender site

ivelasq committed:
Delete folder

ivelasq committed:
Push up site!

ivelasq committed:
Update

Merge remote-tracking branch 'origin/main' into base-r-fns

# Conflicts:
#	_site/blog/bookdown-manuscript/thumbnail.png
#	_site/blog/you-cran-do-it/thumbnail.png
#	_site/index.html
#	_site/listings.json
#	_site/search.json
#	_site/site_libs/bootstrap/bootstrap-icons.css
#	_site/site_libs/bootstrap/bootstrap-icons.woff
#	_site/site_libs/bootstrap/bootstrap.min.css
#	_site/site_libs/bootstrap/bootstrap.min.js
#	_site/site_libs/clipboard/clipboard.min.js
#	_site/site_libs/quarto-html/anchor.min.js
#	_site/site_libs/quarto-html/popper.min.js
#	_site/site_libs/quarto-html/quarto-syntax-highlighting.css
#	_site/site_libs/quarto-html/quarto.js
#	_site/site_libs/quarto-nav/quarto-nav.js
#	_site/site_libs/quarto-search/autocomplete.umd.js
#	_site/site_libs/quarto-search/fuse.min.js
#	_site/site_libs/quarto-search/quarto-search.js
#	_site/sitemap.xml
#	blog/not-so-basic-base-r-functions/index.html
#	talk/blogdown-for-what/thumbnail2.png
#	talk/building-a-blog-with-quarto/thumbnail2.png
#	talk/building-a-blog-with-r/thumbnail2.png
#	talk/git-through-april/thumbnail2.png
#	talk/gt-talk/thumbnail2.png
#	talk/hiking-data/thumbnail2.png
#	talk/intro-to-quarto/thumbnail2.png
#	talk/longitudinal-analysis/thumbnail2.png
#	talk/package-mangement-in-r/index_files/fitvids/fitvids.min.js
#	talk/package-mangement-in-r/index_files/header-attrs/header-attrs.js
#	talk/package-mangement-in-r/thumbnail2.png
#	talk/rladies-allyship/thumbnail2.png
#	talk/using-r-and-other-tools/thumbnail2.png
#	webr-serviceworker.js
#	webr-worker.js

ivelasq committed:
Update site

ivelasq committed:
Update date

ivelasq committed:
Continue post

ivelasq committed:
Add webr install code

ivelasq committed:
Continue post

ivelasq committed:
Continue post

ivelasq committed:
Update extension

ivelasq committed:
Continue webR post

ivelasq committed:
Continue post on webR

ivelasq committed:
Continue webR post
